demo/apisix/conf/apisix.yaml

```diff
@@ -227,5 +228,11 @@ plugin_configs:
               - OPTIONS
         conf:
           - name: TokenRevokedFilter
-            value: '{"enable":"feature"}'
+            value: ''
```

Comment:
what was the enable: feature for?

Reply:
`{"enable":"feature"}` I copied (monkey-copy) it from the example when implementing `TokenRevokedFilter`,  And now after time with Apisix, I realize we don't use it in source code.